### DISCUSSION TITLE
1x vs 2x, behaviour is flipped.

### DISCUSSION THREAD

**Discussion by poetaster** [poetaster/machines-vs-machines-sfos#1] [Ideas]
@cequence wrote: I can also raise a ticket about speed, 1x vs 2x, behaviour is flipped. 1x is the fast speed and 2x is the slow instead of the expected ==> 2x should be the faster setting. Once issues are open I can post a ticket.

**Comment by poetaster** [comment]
Hmmm. When in normal speed, 2x is displayed, click and the alien horde accelerates. In accelerated speed mode, 1x is displayed. Clicking returns to normal speed. I find that intuitive, but one could have 1x displayed when in normal speed and clicking it lead to the display of 2x and at double speed. Clicking 2x would lead to a return to normal and display of 1x. I assume that’s what you mean?

**Reply by cquence** [reply]
Yeah I suppose the way you describe it makes sense, the icon shows the "next state" you will be in when you click it, not the current one. From this perspective it makes sense, I think I got it wrong.

**Reply by poetaster** [reply]
Ok. If you think it's 'ok' in it's current state, I'll consider it a 'won't fix'?

**Reply by cquence** [reply]
Sounds good to me. Thanks a lot!

**Reply by direc85** [reply]
There are two ways to think about the behavior, and they both make sense! I too thought it was flipped at first. Switching the behavior would only confuse "the old players", so personally I would also leave it as-is.
